moshi/src/test/java/com/squareup/moshi/PromoteNameToValueTest.java

```diff
@@ -278,7 +279,7 @@
   @Test public void writerNullValue() throws Exception {
```

Comment:
this can't actually happen in practice because the map adapter doesn't allow null keys, but i'll update the error message, too. be right back.

Reply:
did it.